helm/oauth2-proxy/Chart.yaml

```diff
@@ -22,8 +22,12 @@ sources:
 - https://github.com/oauth2-proxy/oauth2-proxy
 - https://github.com/oauth2-proxy/manifests
 maintainers:
-- email: cedric@desaintmartin.fr
-  name: desaintmartin
+- name: desaintmartin
+  email: cedric@desaintmartin.fr
 - name: tlawrie
+- name: NickMeves
+  email: nick.meves@greenhouse.io
```

Comment:
Nick is no longer with greenhouse, so will need to update this one

@NickMeves Which email do you want to use here?

Reply:
`nicholas.meves@gmail.com`